applications/StructuralMechanicsApplication/tests/structural_response_function_test_factory.py

```diff
@@ -150,12 +143,11 @@ class TestStrainEnergyResponseFunction(StructuralResponseFunctionTestFactory):
     def test_execution(self):
         self.current_model = KratosMultiphysics.Model()
         self._calculate_response_and_gradient()
-        self.assertAlmostEqual(self.value, 0.00606275111915477)
+        self.assertAlmostEqual(self.value, 0.6062751119154768)
```

Comment:
Why did you change this and the below values by a factor 100? Where did this come from?

Reply:
Previously there where two primal parameter files, one for the old version of the strain energy and one for the adjoint workflow with hdf5. 
For some reason they had different load magnitudes. Now as none of the tests is using hdf5 anymore i could use the same primal parameters everywhere, thats why i had to adapt the test results for the old strain energy response.